lib/routes/content.js

```diff
@@ -9,11 +9,9 @@ const content = require('punchcard-content-types');
 const multipart = require('connect-multiparty');
 const uuid = require('uuid');
 const _ = require('lodash');
-
 const utils = require('../utils');
```

Comment:
We've been informally adding this space between npm dependencies and local `require`s. @Snugug should this be a formal decision to keep this space?